### PR TITLE
Fix for tieba.baidu.com

### PR DESCRIPTION
**Pull request by felixonmars**
- Add many more fixes after #6491 landed

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -12125,20 +12125,30 @@ INVERT
...
 .nav_wrap_add_border
 .save_face_bg
 .share_btn_wrapper
+.sub_nav_wrap
+.threadlist_rep_num
+.u-f-item
 
 CSS
+.card_banner {
+    opacity: 50%;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12141]: Hmm interesting, what is this change for? We mostly don't need to poke around the opacity.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -12125,20 +12125,30 @@ INVERT
...
+.f-d-item,
 .left-sec,
+.page-container,
+.pb_footer,
 .sub_nav_wrap,
-.tb_footer,
-.u-f-item {
-    background-color: var(--darkreader-neutral-background) !important;
+.tb_footer {
+    background: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12151]:
```suggestion
    background-image: none !important;
```
Specify it a bit further.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config:12141]: Most banners are too bright for dark theme. Perhaps I should implement it in a different way?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:12141]: Well, we currently uses `filter: brightness(50%) sepia(40%) !important;` to ensure the brightness is down and add  bit of sepia over it to make it more natural.

Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config]: Looks like turning down some brightness is enough here (and looks the same as opacity). Updated.

**Review by Gusted — APPROVED** [review]
All good.

**Pull request merged**